### ISSUE TITLE
No Reviews count as one 0 star view

### ISSUE THREAD

**Issue by LivacoNew** [everyday-as/gmodstore-issues#1271]
### Expected Behavior

No Reviews on a product doesn't display anything, as there isn't any feedback.

### Actual Behavior

On addon cards, it shows 0 stars with (1) indicating one 0 star review has been posted.
![TjCXmp5tWw 1](https://github.com/everyday-as/gmodstore-issues/assets/51921257/514855d6-5c36-454c-818c-1909b1cc71dc)
![ZNQyImijE0 1](https://github.com/everyday-as/gmodstore-issues/assets/51921257/d2a2472c-dd21-44d5-94b6-90ef52a0288f)

### Reproduction Steps

```markdown
1. Make addon
2. Wait the new hidden review timing thing
3. With no reviews, your addons now 0 stars.
```

### Browsers

Firefox 116.0.2 (64-bit)

### Example URL

https://www.gmodstore.com/market/view/fleodon-s-doorbells-ring-with-style

### GmodStore Version

v14.11.1

**Comment by Marcuzz** [comment]
Dev note: this is because of the new 48 hour period where new reviews are not shown to the public - the issue is that the count is bumped even though it shouldn't at this point in time

**Comment by J0sh0nat0r** [comment]
Fixed in v14.12.0
